src/archivematica/dashboard/media/js/misc.js

```diff
@@ -138,4 +138,31 @@ $(document).ready(
                 $(this).closest('.content').hide();
                 $(this).closest('.preview-help-text').children('.preview').show();
               });
+
+      // Enable tooltip for the button which allows users to copy the API key to the clipboard.
+      $('#copy-api-key-button').tooltip();
+
+      // Copy the API key to the clipboard on button click, then update the tooltip and icon accordingly.
+      $('#copy-api-key-button').click(function () {
```

Comment:
You could use chaining here as long as tooltip returns the jQuery object - slightly faster because jQuery doesn't have to look up the same element twice, e.g.:

```js
$('#copy-api-key-button')
  .tooltip()
  .click(function () {
    // ...
  });
```

Reply:
Thank you @sevein! I implemented chaining in https://github.com/artefactual/archivematica/pull/2041/commits/bab337348d3ff0f2e1fae9ec01d13bdb036e0af3 :muscle: